packages/web-platform/web-rsbuild-plugin/tests/basic.test.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+import { createRsbuild } from '@rsbuild/core';
+import { describe, expect, test } from 'vitest';
+import path from 'path';
+import { pluginWebPlatform } from '../dist/index.js';
```

Comment:
```suggestion
import { pluginWebPlatform } from '../src/index.js';
```

Reply:
It needs to reference the product because loader.js uses a relative path and the source code is loader.ts